e2e/atlas/logs_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e,atlas
+
+package atlas_test
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/mongodb/mongocli/e2e/atlas"
+)
+
+func TestAtlasLogs(t *testing.T) {
+	cliPath, err := filepath.Abs("../../bin/mongocli")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	_, err = os.Stat(cliPath)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	atlasEntity := "atlas"
+	logsEntity := "logs"
+	clusterName := ""
+
+	// Deploy a cluster only if there is not one available
+	if !atlas.ExistCluster(cliPath) {
```

Comment:
I thought this was a good idea: deploying a new cluster only if there are no available but now metric is failing probably because of it. It does not create a new cluster because one cluster is running but then the test fails because the cluster has been deleted.
Can we just deploy one cluster in atlas that we use for testing metrics, logs etc and we do not remove it? This would simplify the code and reduce the time to run the test. @gssbzn

Reply:
> Can we just deploy one cluster in atlas that we use for testing metrics
short answer, no, they get removed after 24h
This goes back to to a point I made before, probably you're better off running all this tests depending on the cluster create test
Or both logs and metrics will need to create their own cluster to watch for
if you decide to go for logs and metrics depending on their own cluster I'd advice to start separating these tests in different evg tasks so they can run on parallel